src/runtime/runtime_mimxrt1062_clock.go

```diff
@@ -166,7 +166,7 @@ func initClocks() {
 	nxp.ClockIpLpi2c2.Enable(false) //
 	nxp.ClockIpLpi2c3.Enable(false) //
```

Comment:
ClockIpLpi2c4.Enable() is required.
The same change is required for enablePeripheralClocks().